aten/src/ATen/core/function.h

```diff
@@ -25,6 +25,11 @@ TORCH_API void preoptimizeGraph(std::shared_ptr<Graph>& graph);
 // execution of the function. Method is a wrapper around a
 // underlying Function that also provides a `self` object.
 struct TORCH_API Function {
+  virtual const std::string& doc_string() const {
```

Comment:
I think we can add the doc_string here, make it a default "', and then remove the `no_doc_string`?